internal/operator/client.go

```diff
@@ -27,8 +27,7 @@ type Client interface {
 	DeleteOperatorGroup(ctx context.Context, name string, namespace string) error
 	CreateSubscription(ctx context.Context, data SubscriptionData, namespace string) (*operatorv1alpha1.Subscription, error)
 	DeleteSubscription(ctx context.Context, name string, namespace string) error
-	InstallPlanApprove(namespace string) error
```

Comment:
Removing functions from an interface can be scary.   unless these are internal only?

Reply:
They became internal check the installPlan.go file.

Reply:
Looked up references to both functions.  looks like they had only one caller each - and you've wrapped that into ApproveInstallPlan.
Just to be sure - nothing outside opcap e.g. in Melvin's UI code calls this directly or anything right?

Reply:
https://github.com/search?q=org%3Aopdev+InstallPlanApprove&type=code  

nope.
resolving.